sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample1_RecognizeReceiptsFromFile.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    [LiveOnly]
+    public partial class FormRecognizerSamples
+    {
+        [Test]
+        public async Task RecognizeReceiptsFromFile()
+        {
+            string endpoint = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_ENDPOINT");
+            string apiKey = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_API_KEY");
+
+            #region Snippet:FormRecognizerSample1CreateClient
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            #endregion
+
+            string receiptPath = TestEnvironment.JpgReceiptPath;
+
+            #region Snippet:FormRecognizerSample1RecognizeReceiptFileStream
+            using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
+            {
+                Response<IReadOnlyList<RecognizedReceipt>> receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
+                foreach (var receipt in receipts.Value)
+                {
+                    USReceipt usReceipt = receipt.AsUSReceipt();
+
+                    string merchantName = usReceipt.MerchantName;
+                    DateTime transactionDate = usReceipt.TransactionDate;
+                    IReadOnlyList<USReceiptItem> items = usReceipt.Items;
+                    float subtotal = usReceipt.Subtotal;
+                    float tax = usReceipt.Tax;
+                    float total = usReceipt.Total;
+
+                    Console.WriteLine($"Recognized USReceipt fields:");
+                    Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {usReceipt.MerchantName.Confidence}");
+                    Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {usReceipt.TransactionDate.Confidence}");
+
+                    for (int i = 0; i < items.Count; i++)
+                    {
+                        USReceiptItem item = usReceipt.Items[i];
+                        Console.WriteLine($"    Item {i}:  Name: '{item.Name.Value}', Quantity: '{item.Quantity?.Value}', TotalPrice: '{item.TotalPrice.Value}'");
+                    }
+
+                    Console.WriteLine($"    Subtotal: '{subtotal}', with confidence '{usReceipt.Subtotal.Confidence}'");
+                    Console.WriteLine($"    Tax: '{tax}', with confidence '{usReceipt.Tax.Confidence}'");
```

Comment:
should we include the Tip too?

Reply:
Happy to.  I didn't include all the Merchant fields for brevity - the address is rather long.

Reply:
haha sure.
I mean just looking at a receipt I will expect to "always" see tip. 
the address and other stuff can be omitted here